.github/workflows/claude.yml

```
@@ -0,0 +1,45 @@
+  issue_comment:
+    types: [created]
+  pull_request_review_comment:
+    types: [created]
+  issues:
+    types: [opened, assigned]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  claude:
+    if: false # Temporarily disabled while actively developing
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)